ckanext-unckan/test.ini

```
@@ -8,7 +8,7 @@ use = config:../../ckan/test-core.ini
 
 # Insert any custom config settings to be used when running your extension's
 # tests here. These will override the one defined in CKAN core's test-core.ini
-ckan.plugins = push_errors unckan pdf_view datapusher_plus api_tracking superset announcements
+ckan.plugins = push_errors unckan pdf_view datapusher_plus api_tracking tracking superset announcements
```

```suggestion
ckan.plugins = push_errors unckan pdf_view datapusher_plus tracking api_tracking superset announcements
```
Probar con `tracking` antes de `api_tracking`
Quizas el otro PR externos no sea necesario